recipes/statmorph/meta.yaml

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Nit: you could drop this blank line.

Reply:
Thanks! This should be taken care of.

Reply:
It is. Just GitHub diffs can be dumb sometimes and hence move comments to match the recent PR diff. 😞